src/app/pages/TopicPage/index.styles.jsx

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/react';
+
+const styles = {
+  main: ({ spacings, mq }) =>
+    css({
+      margin: `0 ${spacings.FULL}rem`,
+      [mq.GROUP_2_MIN_WIDTH]: {
+        margin: `0 ${spacings.DOUBLE}rem`,
+      },
+    }),
+  inner: css({
+    maxWidth: '63rem',
```

Comment:
As below, use a constant if one is available

Reply:
https://github.com/bbc/simorgh/pull/10342/files#diff-92057634d5283330a6167da6a5a33b6e5d75b3fed7e09cc1de94f12e797702abL41